tmail-backend/jmap/extensions/src/test/scala/com/linagora/tmail/james/jmap/label/MemoryLabelChangeRepositoryTest.scala

```diff
@@ -0,0 +1,25 @@
+package com.linagora.tmail.james.jmap.label
```

Comment:
Now can we move also the memory implementations into dedicated packages of the `extension-api` module? (I see no reason to leave it in the JMAP code...)

Reply:
Yes we can, but I suggest we take care of that in another PR as this one is getting scary...

Reply:
https://github.com/linagora/tmail-backend/issues/1399